### PR TITLE
Integration of console-iot with master

### PR DESCRIPTION
**Pull request by drcoolsanjeev**
### Type of change

- Enhancement / new feature

### Description

Enable the management of IoT features through Console:

- Visual IoT Screens
- Integration of the IoT screens to Mock Server
- Enable Mock Server to include schema and functionality to support IoT.
- Upgrade Patternfly to the latest release

### Marvel Designs 

https://marvelapp.com/prototype/5gbji22

### Checklist

- [ ] Update/write design documentation in `./documentation/design`
- [ ] Write tests and make sure they pass
- [ ] Update documentation
- [ ] Check RBAC rights for Kubernetes / OpenShift roles
- [ ] Try your changes from Pod inside your Kubernetes and OpenShift cluster, not just locally
- [ ] Reference relevant issue(s) and close them after merging
- [ ] Update CHANGELOG.md

### CODE REVIEW

File: console/console-init/ui/mock-console-server/schema.js [hunk 1]
```diff
@@ -76,22 +76,22 @@ const typeDefs = gql`
     reencrypt
   }
 
-  enum MessagingEndpointType_enmasse_io_v1 {
+  enum MessagingEndpointType_enmasse_io_v1beta2 {
```
[the remark this answers is not in the document]
Comment (drcoolsanjeev) [reply console/console-init/ui/mock-console-server/schema.js]: Replaced v1beta2 by v1 in mockserver schema.js file.

File: console/console-init/ui/src/components/Editor/Editor.tsx [hunk 2]
```diff
@@ -0,0 +1,38 @@
+import React from "react";
```
Comment (k-wall) [inline console/console-init/ui/src/components/Editor/Editor.tsx:1]: Missing licence.

File: console/console-init/ui/src/components/FinishedStep/FinishedStep.tsx [hunk 3]
```diff
@@ -0,0 +1,106 @@
+import React from "react";
```
Comment (k-wall) [inline console/console-init/ui/src/components/FinishedStep/FinishedStep.tsx:1]: Missing licence.

File: console/console-init/ui/src/components/FormAndJson/FormAndJson.tsx [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+}
+const FormAndJson: React.FunctionComponent<IFormAndJsonProps> = ({
+  detail,
+  detailInJson,
+  setDetail,
+  setDetailInJson
+}) => {
+  const [isValid, setIsValid] = useState<boolean>(true);
+  const [showJson, setShowJson] = useState<boolean>(true);
+  //   const setDetail1 = () => {
```
Comment (k-wall) [inline console/console-init/ui/src/components/FormAndJson/FormAndJson.tsx:18]: commented out example code.

**Comment by anukritijha** [comment]
@drcoolsanjeev @ajaypratap003 @rkpattnaik780 This PR has reintroduced warnings related react-popper, which was resolved earlier.

**Comment by jbtrystram** [comment]
I noticed some inconsistencies in the input types for the graphql iot mutations this morning, i want to align those before it's merged

File: console/console-init/ui/src/components/DialogPrompt/DialogPrompt.tsx [hunk 5]
```diff
@@ -42,20 +72,29 @@ export const DialogPrompt: React.FunctionComponent<{}> = () => {
   return (
     <Modal
       id="Dialogue-prompt-modal"
```
[the remark this answers is not in the document]
Comment (ajaypratap003) [reply console/console-init/ui/src/components/DialogPrompt/DialogPrompt.tsx]: @obabec thanks for the observation. One team member has already started working on changing Ids with defined format  across the application.

[the remark this answers is not in the document]
Comment (obabec) [reply console/console-init/ui/src/components/DialogPrompt/DialogPrompt.tsx]: @ajaypratap003 Okay, cool :)

**Comment by vbusch** [comment]
Looking at PR #4926: downgrading the version of react & typescript, I don't believe it will help the build issues in this PR.    
I have compared a successful build from master: https://pipelines.actions.githubusercontent.com/KxWcr3gUsjLAPBH2otl3MWUiuEaoRcGYQy1SDmNP4xdyBJ0SiN/_apis/pipelines/1/runs/6570/signedlogcontent/5?urlExpires=2020-07-13T19%3A17%3A18.9691159Z&urlSigningMethod=HMACV1&urlSignature=absgcxgpIh6v%2FVxuK69b%2F%2F30Id825zMmocrKNxEReX8%3D
With the failing build from this PR: https://pipelines.actions.githubusercontent.com/KxWcr3gUsjLAPBH2otl3MWUiuEaoRcGYQy1SDmNP4xdyBJ0SiN/_apis/pipelines/1/runs/6560/signedlogcontent/5?urlExpires=2020-07-13T15%3A24%3A35.9669749Z&urlSigningMethod=HMACV1&urlSignature=HQGEBv8onh4LEmEe2DhWg1TraSWV33R6EGJo9Al0PcM%3D
But haven't been able to spot any version differences etc. that would account for it passing locally, and passing on master in jenkins.  But failing in here.
@k-wall the error message is `Failed to execute goal com.github.eirslett:frontend-maven-plugin:1.9.1:yarn (yarn test - console) on project console-init: Failed to run task: 'yarn test --watchAll=false' failed. `  Is it possible to get more logging for this?  I am surprised I can't reproduce it locally.

**Pull request merged**